### PR TITLE
docs: add @all-contributors bot

### PR DESCRIPTION
**Pull request by jakebolam**
##### Checklist

- [x] If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request.
- [x] All comments in frontmatter need to be removed. <- not sure what this means
- [x] Performs a useful action through the GitHub API that solves an existing problem for developers: Recognize [all-contributors](https://github.com/all-contributors)
- [x] Is original: for example, it does something not already done by an existing Probot app
- [x] [Has tests](https://github.com/all-contributors/all-contributors-bot/tree/master/test)
- [x] [Has documentation](https://github.com/all-contributors/all-contributors-bot/)
- [x] [Is open source](https://github.com/all-contributors/all-contributors-bot/tree/master/test)
- [x] [Has a license](https://github.com/all-contributors/all-contributors-bot/blob/master/LICENSE)
- [x] [Has a code of conduct](https://github.com/all-contributors/all-contributors-bot/blob/master/CODE-OF-CONDUCT.md)
- [x] Has someone willing to at least minimally maintain them for the near future: @jakebolam

### CODE REVIEW

File: _apps/all-contributors.m [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+slug: allcontributors
+screenshots:
+- https://raw.githubusercontent.com/all-contributors/all-contributors-bot/master/docs/usage.png
+authors:
+- jakebolam
+repository: all-contributors/all-contributors-bot
+stars: 20
+updated: 2019-01-15 07:53:00 UTC
+host: https://gkioebvccg.execute-api.us-east-1.amazonaws.com/
+installations: 2
```
[the remark this answers is not in the document]
Comment (jakebolam) [reply _apps/all-contributors.m]: Stats updated, based on stats endpoint implemented: https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/stats

File: _apps/all-contributors.m [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+installations: 16
+organizations:
+- all-contributors
+- thechangelog
+- tophat
+- react-native-community
+- dowjones
+---
+
+✨ Recognize all contributors, not just the ones who push code. Simple add a contributor with comments on issues or pull requests, e.g. `@all-contributors please add tbenning for design`
```
Comment (JasonEtco) [inline _apps/all-contributors.m]: Small typo:

```suggestion
✨ Recognize all contributors, not just the ones who push code. Simply add a contributor with comments on issues or pull requests, e.g. `@all-contributors please add tbenning for design`
```

**Comment by jakebolam** [comment]
Thanks @JasonEtco :)

**Comment by j-f1** [comment]
Shouldn’t this be `.md` instead of `.m`?

**Comment by JasonEtco** [comment]
It should.. I'm not sure why its working though 🤔 @jakebolam can you open a PR to change that please?

File: _apps/all-contributors.m [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+description: Easily recognize all contributions per the all-contributors spec
+slug: allcontributors
+screenshots:
+- https://raw.githubusercontent.com/all-contributors/all-contributors-bot/master/docs/usage.png
+authors:
+- jakebolam
+repository: all-contributors/all-contributors-bot
+stars: 39
+updated: 2019-01-21 18:48:00 UTC
+host: https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod
```
Comment (jakebolam) [inline _apps/all-contributors.m]: @JasonEtco for the sync to work. Do the stats need to be at `https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/stats/probot` or  is`https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/stats` fine?

Comment (hiimbex) [reply _apps/all-contributors.m]: If you enter `host: https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod` we expect the stats to be at https://gkioebvccg.execute-api.us-east-1.amazonaws.com/prod/probot/stats

Comment (jakebolam) [reply _apps/all-contributors.m]: Thanks, @hiimbex. I've updated the endpoint path to be at the expected location:
https://github.com/all-contributors/all-contributors-bot/commit/a55498562a169d77d734c1c155b1e09339d3a95c

**Comment by jakebolam** [comment]
@JasonEtco @j-f1 great spot! Put up a PR to fix this: https://github.com/probot/probot.github.io/pull/282

**Pull request merged**